arcgis-ios-sdk-samples/Utility network/Trace utility network/TraceUtilityNetworkViewController.swift

```diff
@@ -25,7 +25,7 @@ class TraceUtilityNetworkViewController: UIViewController, AGSGeoViewTouchDelega
     @IBOutlet weak var statusLabel: UILabel!
     @IBOutlet weak var modeControl: UISegmentedControl!
     
-    private let featureServiceURL = URL(string: "https://sampleserver7.arcgisonline.com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer")!
+    private let featureServiceURL = URL(string: "https://sampleserver7.arcgisonline.com/server/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer")!
```

Comment:
The updated service doesn't show network or controllers. Could you verify that?

Reply:
![noshow](https://user-images.githubusercontent.com/9660181/103235770-bd996200-48f7-11eb-90f3-a48c791cf813.png)

It looks like this to me.

Reply:
Also, should we update the URLs in README?

Reply:
Fixed this sample and also updated the URLs in the READMEs!